src/Silex2ServiceProvider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Bugsnag\Silex;
+
+use Bugsnag\Silex\Request\SilexResolver;
+use Pimple\Container;
+use Pimple\ServiceProviderInterface;
+use Silex\Application;
+use Symfony\Component\HttpFoundation\Request;
+
+class Silex2ServiceProvider extends AbstractServiceProvider implements ServiceProviderInterface
+{
+    /**
+     * Registers services on the container.
+     *
+     * @param \Pimple\Container $app
```

Comment:
Yeh, that's right. The silex 2 service provider is actually a pimple service provider. They moved it out to there.